src/utils.ts

```diff
@@ -4,33 +4,25 @@ import fs from 'fs'
 import { logger } from '@libp2p/logger'
 import { nanoid } from 'nanoid'
 import tempWrite from 'temp-write'
+import type { ControllerOptions, IPFSOptions, NodeType } from '.'
```

Comment:
`import type { ControllerOptions, IPFSOptions, NodeType } from './types.js'`

Reply:
`types.ts` has been removed, all the types are in `index.ts` now.  They were previously re-exported by declaring `@typedef` jsdocs in `index.js` importing from `types.js` but that's not necessary any more.  This could certainly import from `index.js` instead of `.` though it works as is cos tsc can resolve `'.'` then strips it out as it's a type import.

```suggestion
import type { ControllerOptions, IPFSOptions, NodeType } from './index.js'
```